pkg/controller/controller.go

```diff
@@ -908,8 +908,8 @@ func (c *controller) pollInstance(serviceClass *v1alpha1.ServiceClass, servicePl
 		ServiceID: serviceClass.OSBGUID,
 		PlanID:    servicePlan.OSBGUID,
 	}
-	if instance.Status.LastOperation != "" {
-		lastOperationRequest.Operation = instance.Status.LastOperation
+	if instance.Status.LastOperation != nil && *instance.Status.LastOperation != "" {
```

Comment:
it is really annoying that we have to do this.  In java we had [isNullOrEmpty](https://google.github.io/guava/releases/21.0/api/docs/com/google/common/base/Strings.html#isNullOrEmpty-java.lang.String-) to avoid repeating this all over.

WDYT of pulling this into a common function 

isNilOrEmpty(*string) bool {...}